consensus/state.go

```diff
@@ -849,6 +893,7 @@ func (cs *ConsensusState) EnterPrecommitWait(height int, round int) {
 	// After `timeoutPrecommit0+timeoutPrecommitDelta*round`, EnterNewRound()
 	go func() {
 		time.Sleep(timeoutPrecommit0 + timeoutPrecommitDelta*time.Duration(round))
+		cs.evsw.FireEvent(types.EventStringTimeoutWait(), cs.RoundStateEvent())
```

Comment:
Should move into EnterNewRound after condition check, because it didn't necessarily timeout (e.g. it could have gone to commit).